app/assets/images/covid-arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="38" height="38" viewBox="0 0 38 38" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M37.9995 19C37.9995 29.4934 29.4929 38 18.9995 38C9.43459 38 1.52044 30.9322 0.194803 21.7343L21.0336 21.7343L13.7809 28.9869L16.6093 31.8153L27.5877 20.8369L29.0019 19.4227L27.5877 18.0085L16.6093 7.03012L13.7809 9.85855L21.6566 17.7343L0.0410156 17.7343C0.692375 7.83098 8.93144 0 18.9995 0C29.4929 0 37.9995 8.50659 37.9995 19Z" fill="#FFE013"/>
```

Comment:
Worth checking that the fill colour is the correct shade - when we switched to the yellow colour, we noticed that the original SVG colour didn't match the new yellow hex code in the CSS.

Reply:
ahh It's not - the covid yellow colour is `#ffe114`